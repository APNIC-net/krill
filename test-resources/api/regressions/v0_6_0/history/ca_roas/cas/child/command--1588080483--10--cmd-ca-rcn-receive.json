{
  "actor": "krill",
  "time": "2020-04-28T13:28:03.364137Z",
  "handle": "child",
  "version": 15,
  "sequence": 10,
  "details": {
    "update_rcvd_cert": [
      "0",
      {
        "asn": "",
        "v4": "192.168.0.0/16",
        "v6": ""
      }
    ]
  },
  "effect": {
    "events": [
      15,
      16,
      17
    ]
  }
}