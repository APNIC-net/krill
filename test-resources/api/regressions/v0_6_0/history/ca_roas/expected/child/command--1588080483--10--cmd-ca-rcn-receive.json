{
  "command": {
    "actor": "krill",
    "time": "2020-04-28T13:28:03.364137Z",
    "handle": "child",
    "version": 15,
    "sequence": 10,
    "details": {
      "update_rcvd_cert": [
        "0",
        {
          "asn": "",
          "v4": "192.168.0.0/16",
          "v6": ""
        }
      ]
    },
    "effect": {
      "events": [
        15,
        16,
        17
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "child",
        "version": 15,
        "details": {
          "certificate_received": [
            "0",
            "77AA69E9571D1DD092F7331CF7157D07B5E8997B",
            {
              "cert": "MIIHAjCCBeqgAwIBAgIUO+L6AGPqhZyMkWlflYKKfxo4/MUwDQYJKoZIhvcNAQELBQAwggItMYICKTCCAiUGA1UEAxOCAhwzMDgyMDEwQTAyODIwMTAxMDBENTcwNTJGM0Y3RTI5OTFFNDIyMUQ1NzREMDg5NDc2QjUyMjQxOTJDRjY3N0M5MzkyMjEzMTRFNUU3QTdBNDhBNDFDRkY4QkFBNUU1ODRGMTlEMEU4MkVDRkQ5RDI5Qjg5MTk3MEZCQzRBNTNDREY4NkY2NjVBQURCODM0RDNCQTVCMzU3RTc4RTc3RUQzNTA3QkJFNUY5NUYxNUQ5NzZBNDcwRjdGNzI1NjBEM0EyNTBFM0YwRTU3QzQ5RkMzNEE4MjI2OUREQjYxNDE0NUM5MjBGNEFCRTI2MjM3RTMzMDYyNDYxMTNGNDBFNDc2NTc2QTQ3QzcxNDYxOUU3RUM2ODQzQkM3NkY3RUVBOThGNjY4MzJFQUYwQzE4ODgwREQ1NzQ5RUFFMTMzRTlENzFDNjYxRUM0QkEyNUNCMTk2MDY1Q0IwNDlFRTMyQzFFMjhEOUIwNDE3OEQwMkUzNUFGNTQ0NEQ4RjVDRDU0QzUyMjNENURENjg4MDk3ODFDMDE2NERDMjYzMkRENjc5ODYxNUE2QTFFNEUyRjZFMjI1MDM1M0ExMjMyNzI2Q0FFQzE3MTgzODEyN0U1REM1OEI0NUMyOUI3MUM1MEYxMDU1MjE2NTBFNTkzMjIwQTU2QTkwQjczNEJDODA4MjVDOURDRjIzQUVFNUUxNzcxRDJFOTAyMDMwMTAwMDEwHhcNMjAwNDI4MTMyMzAzWhcNMjEwNDI4MTMyODAzWjAzMTEwLwYDVQQDEyg3N0FBNjlFOTU3MUQxREQwOTJGNzMzMUNGNzE1N0QwN0I1RTg5OTdCMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxfzcrut62t49w60FaZK5SH4hwuRpgfe/PEPrQiJQ6J+812f+ZL3hxchyMwj5rcfAU0tMVrwKBdeZcfLhM7WsUZ4z11edOK4bCDkdLRUwCOdFp4emOCnQz+OFQNB7gRmGy+vmAe4z2E6MsGKI5JmqFF5g5RfYUPcLOsC0vuq9Kos2/bKuYl7YTUkJmIvxOEkbhiNpTD8yo4u/W+qWLZKRsZgvgTJ49QXYPsD7lXSCBmhypwfpEW6+2eIm5sqaN20RcIknW9ZISrT0KHi2oNdKrJIIGMtjyoeU4BrEcsNvDT9yLxM/Ztlhi7Hf6SL0ftN53SMHiv2LdsdpsArH8jSm7QIDAQABo4ICEDCCAgwwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUd6pp6VcdHdCS9zMc9xV9B7XomXswHwYDVR0jBBgwFoAUbKywsyXpTjVWMCm+YXmGJbjPRHswDgYDVR0PAQH/BAQDAgEGMFkGA1UdHwRSMFAwTqBMoEqGSHJzeW5jOi8vbG9jYWxob3N0L3JlcG8vdGEvMC82Q0FDQjBCMzI1RTk0RTM1NTYzMDI5QkU2MTc5ODYyNUI4Q0Y0NDdCLmNybDA8BggrBgEFBQcBAQQwMC4wLAYIKwYBBQUHMAKGIHJzeW5jOi8vbG9jYWxob3N0L3JlcG8vdGEvdGEuY2VyMIHVBggrBgEFBQcBCwSByDCBxTArBggrBgEFBQcwBYYfcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzBXBggrBgEFBQcwCoZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IubWZ0MD0GCCsGAQUFBzANhjFodHRwczovL2xvY2FsaG9zdDozMDAwL3Rlc3QtcnJkcC9ub3RpZmljYXRpb24ueG1sMBgGA1UdIAEB/wQOMAwwCgYIKwYBBQUHDgIwHgYIKwYBBQUHAQcBAf8EDzANMAsEAgABMAUDAwDAqDANBgkqhkiG9w0BAQsFAAOCAQEATurNZ188yJutuwLrTrzGeZf2JBK59SOwdiX7aPaqKNZEt0S2i+xs9cdECmdCTjG7YzDmehir6I5BXL7f0wyBR5aI/ygMEhWp1UQtGeofY9c+eCqDA8WLgAY/mR/eCb2mYce+fshbr0fzuR9xQIE3IlNvLduDcw780T1Egw1uAOV30WRE7xkkepQPxKP4LvGQrs3Zu8PKNs60gKQw1MlQE3AYPNez+6Mt+mqjf/bQr04KiNUNxkzT4Dlp+NqL0xJbluxYKsD7Bwta9lwTBPi9rjWr3AOkMPtja29R1k1EAlx73KHsZzMWR7ONeaPZMIvR+Qbv+eZTBvvO112fjPHNVg==",
              "uri": "rsync://localhost/repo/ta/0/77AA69E9571D1DD092F7331CF7157D07B5E8997B.cer",
              "resources": {
                "asn": "",
                "v4": "192.168.0.0/16",
                "v6": ""
              }
            }
          ]
        }
      },
      {
        "id": "child",
        "version": 16,
        "details": {
          "roas_updated": [
            "0",
            {
              "updated": {},
              "removed": {
                "2001:db8::/32-48 => 64496": {
                  "revocation": {
                    "serial": "647653598336635334152612034593139490575131039176",
                    "expires": "2021-04-28T13:28:00Z"
                  },
                  "hash": "f09935b85169abd77f1d1399ea2e2b86703eb3e494ace6e02aae3396ffd36a31"
                }
              }
            }
          ]
        }
      },
      {
        "id": "child",
        "version": 17,
        "details": {
          "object_set_updated": [
            "0",
            {
              "77AA69E9571D1DD092F7331CF7157D07B5E8997B": {
                "number": 4,
                "revocations_delta": {
                  "added": [
                    {
                      "serial": "647653598336635334152612034593139490575131039176",
                      "expires": "2021-04-28T13:28:00Z"
                    },
                    {
                      "serial": "96405456178813098272299971010248005161268064342",
                      "expires": "2020-04-29T13:28:01.262670Z"
                    }
                  ],
                  "dropped": []
                },
                "manifest_info": {
                  "name": "77AA69E9571D1DD092F7331CF7157D07B5E8997B.mft",
                  "current": {
                    "content": "MIIJRAYJKoZIhvcNAQcCoIIJNTCCCTECAQMxDzANBglghkgBZQMEAgEFADCBmwYLKoZIhvcNAQkQARqggYsEgYgwgYUCAQQYDzIwMjAwNDI4MTMyMzAzWhgPMjAyMDA0MjkxMzI4MDNaBglghkgBZQMEAgEwUzBRFiw3N0FBNjlFOTU3MUQxREQwOTJGNzMzMUNGNzE1N0QwN0I1RTg5OTdCLmNybAMhAGt4bHkF79vELsgWQoS75sIQeSZdyN20GeD6t8OHCxltoIIGyzCCBscwggWvoAMCAQICFD8V8s/FbRyU02RHHij7D79tGnUtMA0GCSqGSIb3DQEBCwUAMDMxMTAvBgNVBAMTKDc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IwHhcNMjAwNDI4MTMyMzAzWhcNMjAwNTA1MTMyODAzWjCCAi0xggIpMIICJQYDVQQDE4ICHDMwODIwMTBBMDI4MjAxMDEwMENFMzdEODVBQTY5QzA4MzU5NDNFNDA1QkI5MDRGODhBQ0FGNTZBRUJDOTNEODdCRUY1Q0Y5NzJEOUQzMzcyNDYzMjI0NDVFNzU4QkFEMkY5NDBCQ0M3MUM0RDY2RTFDMTFFNzBCQjgyMTU3OUJGQTFGMUI4MjM3NEU3MDlDNzdENjQwMUFDRDg5NUNBQzM0QUEyRUQ3RUVGMkI0RThEQUU3ODU2Qzg4QTM1OTdFRjgzOENBM0FEOTQ3RDI0NUE3NDAwQkFDRTlDRTE1MzU1RTkzMjZERDUzRTk0NzUyMDExRjNCM0Y4QTI2MUI0NzJDMjhGQzEyNTAxMTQyODIxQ0ZGNUI4NEM1RkE1NTg0QzE3RUZDNjRCMjcxNERFMTk4NDY5QjQ3RjVEOTg5NjdEM0UzRjg0OTFBODE1Mzk1QUYwNTM4OTlENDFEODlEMjlGRDc5OUY3QTY1NjE1QkVCQTFBRDg0MzAwMEQ3ODBFMzIxRTMwMzc4MkVGRUUyRDM4MjYwNTdCRkEyQUYwRTA3NDVGNDNBQUZERkI5NTg1RDY0QzcyNzg1MDU5MTMzMEZDQUZDNTRBOUMyQURDQTZEQjI3QjU2REExODhBOUVBNjhGNkM5OTk4RTExQjFFRjE5RDBBOTNEQ0Y2QjMzNzkzN0VGQTExMEM0NzBGQjlGNjkzMDIwMzAxMDAwMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAM432FqmnAg1lD5AW7kE+IrK9WrryT2HvvXPly2dM3JGMiRF51i60vlAvMccTWbhwR5wu4IVeb+h8bgjdOcJx31kAazYlcrDSqLtfu8rTo2ueFbIijWX74OMo62UfSRadAC6zpzhU1XpMm3VPpR1IBHzs/iiYbRywo/BJQEUKCHP9bhMX6VYTBfvxksnFN4ZhGm0f12Yln0+P4SRqBU5WvBTiZ1B2J0p/XmfemVhW+uhrYQwANeA4yHjA3gu/uLTgmBXv6KvDgdF9Dqv37lYXWTHJ4UFkTMPyvxUqcKtym2ye1baGIqepo9smZjhGx7xnQqT3PazN5N++hEMRw+59pMCAwEAAaOCAdUwggHRMB0GA1UdDgQWBBRFmY6yzC8cQ+HpkMFAwezB4SHKtjAfBgNVHSMEGDAWgBR3qmnpVx0d0JL3Mxz3FX0HteiZezAOBgNVHQ8BAf8EBAMCB4AwXAYDVR0fBFUwUzBRoE+gTYZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IuY3JsMGQGCCsGAQUFBwEBBFgwVjBUBggrBgEFBQcwAoZIcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby90YS8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IuY2VyMGcGCCsGAQUFBwELBFswWTBXBggrBgEFBQcwC4ZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IubWZ0MBgGA1UdIAEB/wQOMAwwCgYIKwYBBQUHDgIwIQYIKwYBBQUHAQcBAf8EEjAQMAYEAgABBQAwBgQCAAIFADAVBggrBgEFBQcBCAEB/wQGMASgAgUAMA0GCSqGSIb3DQEBCwUAA4IBAQBtZYI3U/mfT96zrA1CyGwGw4ZrJZcLrYFndttuuEjrMpq5UYkKtSzStrcKfdvx5lTUsBaBJOXJZyZnkq1GqtOnCOjpO1drh5cloAmuyDWcsEm5e+DRCXXjofsTA91rg0RwzHKHL6eJBETyOsZTphnLaNl/ggry+7/4qWlLVU2Y5dmkr9zQc10Rpnpr3mXsPbDR+knBcMqbeVJ8JDyVU+gaDNP8neIyC6E+iQHArRpAqwDbFu7eQ6xVnE38aKnFtuNkkTs0U8svy4E4U//dX1/cMB1Nq7apLXpBQOt/tKPhJ+YZ8PLmPG4sRU4IA3FIGcYfRBXAfts49Qzy/sDEoxONMYIBrDCCAagCAQOAFEWZjrLMLxxD4emQwUDB7MHhIcq2MA0GCWCGSAFlAwQCAQUAoGswGgYJKoZIhvcNAQkDMQ0GCyqGSIb3DQEJEAEaMBwGCSqGSIb3DQEJBTEPFw0yMDA0MjgxMzI4MDNaMC8GCSqGSIb3DQEJBDEiBCCGzbHJM3F931EexICajDRA3HBahf5plRM0kFp4LVawGDANBgkqhkiG9w0BAQEFAASCAQArvc57lY5k5gzgATay9xThbK0M4LWa7rUEWUYq5HVL0kH2heMM2GV8ohg0nGSwUlTsukRYO96JvSZgOAW2Zi6f60PYxUC/kwx2J9wxWDNBq5z7Uyc64O/IUjmUj2+TvPHx6fNZE7Ki41VQUlgd9SYlHlnKmwUgIXQvjzG3BLAxP6vzHnNmiAzwx7FZN5FDnENNy8AnrH0vvzcbShgKA8xp1da3//YkBcIZ6NH3lpqgw0C8DlcICabPmCNXrpcW53K/XR1cSX3CWxEt+1Cb/47pOZp9DVxhOcL5MYpOZEc9ICh5M7c6axwEhWH2ITBLp9lK8HJvOK+0vkxG1ahkBG7Y",
                    "serial": "360155886085140926952478977317316240252998415661",
                    "expires": "2020-04-29T13:28:03.368375Z"
                  },
                  "next_update": "2020-04-29T13:28:03.368375Z",
                  "old": "6215dc53d6a4afa8176476c278865d190ea615126bf209fc3e16cd3eba3a2fb5"
                },
                "crl_info": {
                  "name": "77AA69E9571D1DD092F7331CF7157D07B5E8997B.crl",
                  "current": {
                    "content": "MIICdDCCAVwCAQEwDQYJKoZIhvcNAQELBQAwMzExMC8GA1UEAxMoNzdBQTY5RTk1NzFEMUREMDkyRjczMzFDRjcxNTdEMDdCNUU4OTk3QhcNMjAwNDI4MTMyMzAzWhcNMjAwNDI5MTMyODAzWjCBwzAlAhReip1HQnvrc6dYnQ9km4jIZpor8xcNMjAwNDI5MTMyODAwWjAlAhQKRJZM2i5wDkt5H1Jq8DNx+7XqGBcNMjEwNDI4MTMyODAwWjAlAhQqcSP5Ph+YH4ql3GDfJ3+KwIYsvRcNMjAwNDI5MTMyODAwWjAlAhRxccqxVgrtdnhgzpqf3uoK2BdVyBcNMjEwNDI4MTMyODAwWjAlAhQQ4vhc0QzC5zV/uJEM2YW5UaCwVhcNMjAwNDI5MTMyODAxWqAvMC0wHwYDVR0jBBgwFoAUd6pp6VcdHdCS9zMc9xV9B7XomXswCgYDVR0UBAMCAQQwDQYJKoZIhvcNAQELBQADggEBAC3Pdbq9DpvPjXBcNUp1TTtiq4KF577SEvLnV7bMdzQmkEJwd8IebawPMGJhZx7ecDFo5NUG4m1Xnfs7cMx5KPq2tYU+Mxpi27JRyNg4QpJCw93utTFMQFvlfgfTtsPhSnVcVwPnJn0l2RxxyJdez5YxYyWVU2Yx/YVVECFc2mPBgbtff/4CW1w3VvL5x8wooafJHsfSUqW9nJ0U5BnjG9+Au38AlNLqZ8IHgX3ysap5mPIFDAGZTBv2yiXFDT9niSeb3emFZO9lHV3dCXCeCRSyaZg91xuI2MU63N+ds2MwwhzoD+MGOWn/eoGa5VRcPgouLeH1qsug61GRNpI6cUk=",
                    "serial": "4",
                    "expires": "2020-04-29T13:28:03.364894Z"
                  },
                  "old": "47257110613b644f12d80684ab476af238ff24f3e4ad31613f4891716bd4cfa2"
                },
                "objects_delta": {
                  "ca_repo": "rsync://localhost/repo/child/0",
                  "added": [],
                  "updated": [
                    {
                      "name": "77AA69E9571D1DD092F7331CF7157D07B5E8997B.crl",
                      "object": {
                        "content": "MIICdDCCAVwCAQEwDQYJKoZIhvcNAQELBQAwMzExMC8GA1UEAxMoNzdBQTY5RTk1NzFEMUREMDkyRjczMzFDRjcxNTdEMDdCNUU4OTk3QhcNMjAwNDI4MTMyMzAzWhcNMjAwNDI5MTMyODAzWjCBwzAlAhReip1HQnvrc6dYnQ9km4jIZpor8xcNMjAwNDI5MTMyODAwWjAlAhQKRJZM2i5wDkt5H1Jq8DNx+7XqGBcNMjEwNDI4MTMyODAwWjAlAhQqcSP5Ph+YH4ql3GDfJ3+KwIYsvRcNMjAwNDI5MTMyODAwWjAlAhRxccqxVgrtdnhgzpqf3uoK2BdVyBcNMjEwNDI4MTMyODAwWjAlAhQQ4vhc0QzC5zV/uJEM2YW5UaCwVhcNMjAwNDI5MTMyODAxWqAvMC0wHwYDVR0jBBgwFoAUd6pp6VcdHdCS9zMc9xV9B7XomXswCgYDVR0UBAMCAQQwDQYJKoZIhvcNAQELBQADggEBAC3Pdbq9DpvPjXBcNUp1TTtiq4KF577SEvLnV7bMdzQmkEJwd8IebawPMGJhZx7ecDFo5NUG4m1Xnfs7cMx5KPq2tYU+Mxpi27JRyNg4QpJCw93utTFMQFvlfgfTtsPhSnVcVwPnJn0l2RxxyJdez5YxYyWVU2Yx/YVVECFc2mPBgbtff/4CW1w3VvL5x8wooafJHsfSUqW9nJ0U5BnjG9+Au38AlNLqZ8IHgX3ysap5mPIFDAGZTBv2yiXFDT9niSeb3emFZO9lHV3dCXCeCRSyaZg91xuI2MU63N+ds2MwwhzoD+MGOWn/eoGa5VRcPgouLeH1qsug61GRNpI6cUk=",
                        "serial": "4",
                        "expires": "2020-04-29T13:28:03.364894Z"
                      },
                      "old": "47257110613b644f12d80684ab476af238ff24f3e4ad31613f4891716bd4cfa2"
                    },
                    {
                      "name": "77AA69E9571D1DD092F7331CF7157D07B5E8997B.mft",
                      "object": {
                        "content": "MIIJRAYJKoZIhvcNAQcCoIIJNTCCCTECAQMxDzANBglghkgBZQMEAgEFADCBmwYLKoZIhvcNAQkQARqggYsEgYgwgYUCAQQYDzIwMjAwNDI4MTMyMzAzWhgPMjAyMDA0MjkxMzI4MDNaBglghkgBZQMEAgEwUzBRFiw3N0FBNjlFOTU3MUQxREQwOTJGNzMzMUNGNzE1N0QwN0I1RTg5OTdCLmNybAMhAGt4bHkF79vELsgWQoS75sIQeSZdyN20GeD6t8OHCxltoIIGyzCCBscwggWvoAMCAQICFD8V8s/FbRyU02RHHij7D79tGnUtMA0GCSqGSIb3DQEBCwUAMDMxMTAvBgNVBAMTKDc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IwHhcNMjAwNDI4MTMyMzAzWhcNMjAwNTA1MTMyODAzWjCCAi0xggIpMIICJQYDVQQDE4ICHDMwODIwMTBBMDI4MjAxMDEwMENFMzdEODVBQTY5QzA4MzU5NDNFNDA1QkI5MDRGODhBQ0FGNTZBRUJDOTNEODdCRUY1Q0Y5NzJEOUQzMzcyNDYzMjI0NDVFNzU4QkFEMkY5NDBCQ0M3MUM0RDY2RTFDMTFFNzBCQjgyMTU3OUJGQTFGMUI4MjM3NEU3MDlDNzdENjQwMUFDRDg5NUNBQzM0QUEyRUQ3RUVGMkI0RThEQUU3ODU2Qzg4QTM1OTdFRjgzOENBM0FEOTQ3RDI0NUE3NDAwQkFDRTlDRTE1MzU1RTkzMjZERDUzRTk0NzUyMDExRjNCM0Y4QTI2MUI0NzJDMjhGQzEyNTAxMTQyODIxQ0ZGNUI4NEM1RkE1NTg0QzE3RUZDNjRCMjcxNERFMTk4NDY5QjQ3RjVEOTg5NjdEM0UzRjg0OTFBODE1Mzk1QUYwNTM4OTlENDFEODlEMjlGRDc5OUY3QTY1NjE1QkVCQTFBRDg0MzAwMEQ3ODBFMzIxRTMwMzc4MkVGRUUyRDM4MjYwNTdCRkEyQUYwRTA3NDVGNDNBQUZERkI5NTg1RDY0QzcyNzg1MDU5MTMzMEZDQUZDNTRBOUMyQURDQTZEQjI3QjU2REExODhBOUVBNjhGNkM5OTk4RTExQjFFRjE5RDBBOTNEQ0Y2QjMzNzkzN0VGQTExMEM0NzBGQjlGNjkzMDIwMzAxMDAwMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAM432FqmnAg1lD5AW7kE+IrK9WrryT2HvvXPly2dM3JGMiRF51i60vlAvMccTWbhwR5wu4IVeb+h8bgjdOcJx31kAazYlcrDSqLtfu8rTo2ueFbIijWX74OMo62UfSRadAC6zpzhU1XpMm3VPpR1IBHzs/iiYbRywo/BJQEUKCHP9bhMX6VYTBfvxksnFN4ZhGm0f12Yln0+P4SRqBU5WvBTiZ1B2J0p/XmfemVhW+uhrYQwANeA4yHjA3gu/uLTgmBXv6KvDgdF9Dqv37lYXWTHJ4UFkTMPyvxUqcKtym2ye1baGIqepo9smZjhGx7xnQqT3PazN5N++hEMRw+59pMCAwEAAaOCAdUwggHRMB0GA1UdDgQWBBRFmY6yzC8cQ+HpkMFAwezB4SHKtjAfBgNVHSMEGDAWgBR3qmnpVx0d0JL3Mxz3FX0HteiZezAOBgNVHQ8BAf8EBAMCB4AwXAYDVR0fBFUwUzBRoE+gTYZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IuY3JsMGQGCCsGAQUFBwEBBFgwVjBUBggrBgEFBQcwAoZIcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby90YS8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IuY2VyMGcGCCsGAQUFBwELBFswWTBXBggrBgEFBQcwC4ZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IubWZ0MBgGA1UdIAEB/wQOMAwwCgYIKwYBBQUHDgIwIQYIKwYBBQUHAQcBAf8EEjAQMAYEAgABBQAwBgQCAAIFADAVBggrBgEFBQcBCAEB/wQGMASgAgUAMA0GCSqGSIb3DQEBCwUAA4IBAQBtZYI3U/mfT96zrA1CyGwGw4ZrJZcLrYFndttuuEjrMpq5UYkKtSzStrcKfdvx5lTUsBaBJOXJZyZnkq1GqtOnCOjpO1drh5cloAmuyDWcsEm5e+DRCXXjofsTA91rg0RwzHKHL6eJBETyOsZTphnLaNl/ggry+7/4qWlLVU2Y5dmkr9zQc10Rpnpr3mXsPbDR+knBcMqbeVJ8JDyVU+gaDNP8neIyC6E+iQHArRpAqwDbFu7eQ6xVnE38aKnFtuNkkTs0U8svy4E4U//dX1/cMB1Nq7apLXpBQOt/tKPhJ+YZ8PLmPG4sRU4IA3FIGcYfRBXAfts49Qzy/sDEoxONMYIBrDCCAagCAQOAFEWZjrLMLxxD4emQwUDB7MHhIcq2MA0GCWCGSAFlAwQCAQUAoGswGgYJKoZIhvcNAQkDMQ0GCyqGSIb3DQEJEAEaMBwGCSqGSIb3DQEJBTEPFw0yMDA0MjgxMzI4MDNaMC8GCSqGSIb3DQEJBDEiBCCGzbHJM3F931EexICajDRA3HBahf5plRM0kFp4LVawGDANBgkqhkiG9w0BAQEFAASCAQArvc57lY5k5gzgATay9xThbK0M4LWa7rUEWUYq5HVL0kH2heMM2GV8ohg0nGSwUlTsukRYO96JvSZgOAW2Zi6f60PYxUC/kwx2J9wxWDNBq5z7Uyc64O/IUjmUj2+TvPHx6fNZE7Ki41VQUlgd9SYlHlnKmwUgIXQvjzG3BLAxP6vzHnNmiAzwx7FZN5FDnENNy8AnrH0vvzcbShgKA8xp1da3//YkBcIZ6NH3lpqgw0C8DlcICabPmCNXrpcW53K/XR1cSX3CWxEt+1Cb/47pOZp9DVxhOcL5MYpOZEc9ICh5M7c6axwEhWH2ITBLp9lK8HJvOK+0vkxG1ahkBG7Y",
                        "serial": "360155886085140926952478977317316240252998415661",
                        "expires": "2020-04-29T13:28:03.368375Z"
                      },
                      "old": "6215dc53d6a4afa8176476c278865d190ea615126bf209fc3e16cd3eba3a2fb5"
                    }
                  ],
                  "withdrawn": [
                    {
                      "name": "323030313a6462383a3a2f33322d3438203d3e203634343936.roa",
                      "hash": "f09935b85169abd77f1d1399ea2e2b86703eb3e494ace6e02aae3396ffd36a31"
                    }
                  ]
                }
              }
            }
          ]
        }
      }
    ]
  }
}